views/src/Pages/ForgotPassword.jsx

```diff
@@ -0,0 +1,208 @@
+import React, { useEffect, useState, useRef } from "react";
+import {
+  Stepper,
+  Button,
+  Group,
+  TextInput,
+  Container,
+  PinInput,
+} from "@mantine/core";
+import axios from "axios";
+
+import { baseUrl } from "../API/index";
+
+
+
+function ForgotPassword() {
+  const [active, setActive] = useState(0);
+  const [email, setEmail] = useState("");
+  const [emailSubmitted, setEmailSubmitted] = useState(false);
+  const [otpValidated, setOtpValidated] = useState(false);
+  const errorMsg = useRef(null);
```

Comment:
We don't need to use useRef here. useState hook is a better option for this

Reply:
Switched error display infrastructure to a state array as opposed to directly writing into the DOM